README.md

```diff
@@ -52,9 +54,15 @@ OGame-clone browser based game engine.
 1. Move source files of the project to your webserver's directory.
 1. Install PHP dependencies.
     - ``composer install --no-dev``
-1. Run installation wizard: http://your_server_address:port/install
+1. Run installation wizard: ``http(s)://<your_server_address>:<port>/install``
 1. Remove ``install/`` directory
 
+### Docker
+
+1. Prepare config file ``mkdir docker && cp config.php docker && cp includes/constants.php docker && chown 33:33 docker/*.php``
```

Comment:
Would it make sense to add an empty `/docker` directory (with eg. a `.gitkeep` file) directly to the repo? This would simplify this step (just a bit).

Reply:
I don't think the `docker/` directory should go into version control. The files in this directory are the data and configuration persisted while the service is running. If we want to keep this directory with `.gitkeep`, we need to add other files below it to `.gitignore`. An easier way is to add it to `.gitignore` and `.dockerignore`.